admin/javascript/src/components/grid-field/row.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SilverStripeComponent from 'silverstripe-component';
+
+class GridFieldRowComponent extends SilverStripeComponent {
+
+  render() {
+    return (
+      <li className="grid-field-row-component [ list-group-item ]">{this.props.children}</li>
```

Comment:
We wanted the view to be flexible for all gridfield type views as well, so they have the ability to become responsive without having to be forced to remove data. The idea was that the most important secondary data could sit below the title inline (dates, user, type...). 

@chillu so do you think its better to display all fields in the one cell for smaller devices, it would make presentation easier but then it doesn't make using tables as semantic for mobile anyway.